src/components/blockHelpers.tsx

```diff
@@ -14,3 +14,18 @@ export const ContentWrapper = styled('div')({
     padding: '0 1rem',
   },
 })
+
+export const MaxWidthContainerComponent = styled('div')({
```

Comment:
isn't this _almost_ the same as `ContentWrapper` 10 lines above? :) any particular reason for not reusing that?